packages/sit-onyx/src/components/OnyxIconButton/OnyxIconButton.vue

```diff
@@ -24,6 +24,7 @@ const emit = defineEmits<{
   <button
     class="onyx-icon-button"
     :aria-label="props.label"
+    :title="props.label"
```

Comment:
question: shouldn't the icon itself also have a description label? 
could be a follow-up task how to approach it, also if we want to provide e.g. default labels for the onyx icons if that is even possible ^^

Reply:
The issue is that icons not necessarily have a single use-case or meaning. The meaning might even be different based on the context it's used in. Therefore, it's preferably to force the user to set a context specific label and to always aria-hide the icon.

Reply:
sounds good :) 
should we then add a ticket to add `label` to OnyxIconProps?

Reply:
No, I think the icon should always rather be used in a context where there is some label or description on the parent or accompanying element

Reply:
ok :) hope UX will consider it ^^

Reply:
in the past there were examples e.g. in the context of a headline status icon or the table, but then the projects devs can still wrap it in a tooltip ^^